### ISSUE TITLE
Setting options for scrolling

### ISSUE THREAD

**Issue by eorroe** [titoBouzout/BufferScroll#18]
I notice that `"typewriter_scrolling": true` works only once you type something on the line.

Can there be another option that keeps the line vertically centered where the cursor is when scrolling with the cursor (meaning going up and down with the cursor)

Like this [typewriter](https://packagecontrol.io/packages/Typewriter) package

I deleted that package because it does not allow using the mouse to select where I want my cursor to move to. (can't issue that because I believe that's the point of that feature in the package)

To see what I'm talking about on that package, you would have to enable: `"typewriter_mode_scrolling" : true`

**Comment by FichteFoll** [comment]
https://github.com/vihangm/ScrollOff does this by adding a delay to the scroll.

https://github.com/SublimeText/ScrollOffset utilizes some mouse event API utility that make it ignore mouse inputs completely, if installed.

**Comment by eorroe** [comment]
Crap none of those are under package control

**Comment by FichteFoll** [comment]
https://packagecontrol.io/packages/ScrollOffset is, but currently marked for ST2-only (which it doesn't need to).

The other one was submitted but has not yet been approved in https://github.com/wbond/package_control_channel/pull/4144, where you can potentially find more context.

**Comment by titoBouzout** [comment]
done

**Comment by eorroe** [comment]
Thanks. But if someone issues something about that it should probably be a setting.

**Comment by titoBouzout** [comment]
okei done :)

**Comment by eorroe** [comment]
Damn your fast. I replied late, because I went to sleep.

**Comment by eorroe** [comment]
I don't want to open a new issue, but is it suppose to keep it open after you close the groups:

![image](https://camo.githubusercontent.com/842789a1d7559dbdbae75e1955abd8647ddd4a43/687474703a2f2f646c2e64726f70626f782e636f6d2f752f393330333534362f5375626c696d65546578742f42756666657253636f6c6c2f73796e632d7363726f6c6c2e706e67)

Like if I press ctrl+alt+1 to go back to one full view it keeps all 3 open?

**Comment by eorroe** [comment]
I know this is probably getting annoying af. But another thing `"synch_marks": true` doesn't work with this awesome package: [check it out](https://packagecontrol.io/packages/MarkAndMove)

**Comment by titoBouzout** [comment]
It works, please open a new issue with correct title and information since this is starting to be confused.
